lib/ui/text_form_field.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/material.dart' as M;
+
+class TextFormField extends M.TextFormField {
+  TextFormField({
+    bool obscureText = false,
+    String placeholder,
+    String Function(String) validator,
+    void Function(String) onChanged,
+  }) : super(
+          obscureText: obscureText,
+          validator: validator,
+          onChanged: onChanged,
+          decoration: M.InputDecoration(
```

Comment:
I thin u could use my input decoration from styles.dart, maybe transform it into the function, which will get placeholder as a parameter. It fits the design even now, but u could improve if u want and it'll look even better

Reply:
This decoration, for now, is used only here.
So I think there's no reason to move it to somewhere 'shared'.
Also, this Widget is very small, so it won't cause the code readability (as it was in main, for example, =) )

Reply:
`This decoration, for now, is used only here.` - yes. I think it's because it is "our textField", that will be the same for each textField in the app, right?
`So I think there's no reason to move it to somewhere 'shared'.` - nono, I meant to import my style from shared and apply it here, like `decoration: textInputDecoration.copyWith(hintText: 'Comment')`, with `import 'package:innopolis_feedback/shared/styles.dart';`
And this will even reduce the size of widget🙂

Reply:
ok, will see what can I do with that

Reply:
Deleted `styles.dart` because now it's not used